Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:lts-alpine
+
+RUN mkdir -p /usr/src/app
+
+WORKDIR /usr/src/app
+
+COPY . . 
+
+RUN npm run build
+
+RUN npm install redis@0.8.1 && \
+    npm install pg@4.1.1 && \
+    npm install memcached@2.2.2 && \
+    npm install aws-sdk@2.738.0 && \
+    npm install rethinkdbdash@2.3.31
```

Comment:
I made some updates regarding installing dependencies (see #11) to make this step easier and I think you should be fine with just `npm run build` here.
I will try adding old document store adapters like aws, rethinkdb, memcached, etc. back later, however they should still be available to use after `npm build run`.